minifi-commons/minifi-commons-schema/src/main/java/org/apache/nifi/minifi/commons/schema/RemoteProcessGroupSchema.java

```diff
@@ -151,7 +152,7 @@ public String getComment() {
         return comment;
     }
 
-    public String getUrl() {
+    public String getUrls() {
         return url;
```

Comment:
Super minor and I honestly didn't look too close to see if it makes sense, but would it be beneficial at all to rename `url` to `urls`, too?